Nodejs/Product/Nodejs/Debugger/Commands/SetBreakpointCommand.cs

```diff
@@ -61,12 +61,9 @@ public SetBreakpointCommand(int id, NodeModule module, NodeBreakpoint breakpoint
             if (_module != null) {
                 _arguments["type"] = "scriptId";
                 _arguments["target"] = _module.Id;
-            } else if (remote) {
+            } else {
                 _arguments["type"] = "scriptRegExp";
                 _arguments["target"] = GetCaseInsensitiveRegex(_position.FileName);
```

Comment:
Split the logic to avoid introducing regressions for the remote case. Updated the script to convert the entire path to a regexp instead of just the file name This also resolved the phantom breakpoint issue I was seeing earlier.

Testing on various node versions < 6.0 now.